sql/updates/world/3.3.5/2016_11_16_00_world.sql

```diff
@@ -0,0 +1,2 @@
+-- Adds a reference to the spell_vael_burning_adrenaline script for Burning Adrenaline (Vael Fight)
+INSERT INTO `world`.`spell_script_names` (`spell_id`, `ScriptName`) VALUES ('18173', 'spell_vael_burning_adrenaline');
```

Comment:
Ahhh, ok. Sorry, I'm new to TC contributions so I guess I should have looked more at the format of the SQL people were submitting.